docs/EVE-LOGGING.md

```diff
@@ -16,6 +16,10 @@ The current log pipeline can be vertically split into the following three stages
 
 All containers will now log directly to stdout, which are then picked up by containerd. Containerd then forwards the logs received to memlogd service that maintains a circular buffer of fixed size for storing log messages. Memlogd provides a way (via unix domain sockets) to read the logs present in it’s buffer and also stream the incoming logs. EVE currently uses imemlogd plugin for rsyslogd (developed internally) to stream logs from memlogd service.
 
+The imemlogd plugin looks for json messages in the log and then skips the preceeding bytes of log if found. This is done to make it possible for rsyslogd to parse the json message and extract fields from message and also add more fields to the existing message. Imemlogd plugin also puts a default tag name of the incoming container name into each of the log messages. For example logs sent to stdout from pillar container are tagged with "pillar.out" and those logs sent to stderr are tagged with "pillar.err" by default. Log messages that are successfully parsed by rsyslogd in json format will be modfied to have some of the existing values changed and some new key-value pairs added. For example rsyslogd adds the current boot image partition to json formatted logs.
+
+Logs coming from pillar container that cannot be json parsed or truncated will have their source field set to pillar.out/pillar.err in Kibana. It is at times useful to look for pillar.out/pillar.err as source match criteria in Kibana.
```

Comment:
Kibana use is part of the controller hence shouldn't be in EVE. Makes sense to say that the source field in the logging API will be set to pillar.out/pillar.err  i.e., s/Kibana/lEVE log API/

Reply:
I'll change.

Reply:
Hey @gkodali-zededa can you please update this as per @eriknordmark recommendation?